ikasaneip/security/db/src/main/java/org/ikasan/security/dao/HibernateSecurityDao.java

```diff
@@ -329,6 +329,12 @@ public void deletePolicy(Policy policy)
         this.getHibernateTemplate().delete(policy);
     }
 
+    @Override
```

Comment:
How come there is only Delete  RoleModule, no add ?

Reply:
as discussed @andrzej-majewski we do not explicitly need to save RoleModule entities.